lms/static/sass/search/_search.scss

```diff
@@ -0,0 +1,174 @@
+.search-bar,
+.course-index .search-bar,
+.dashboard .search-bar {
+
+  @include box-sizing(border-box);
+  position: relative;
+  padding: 5px;
+  font-family: $sans-serif;
+
+  form > div {
+    position: relative;
+  }
+
+  .search-field {
+    @include box-sizing(border-box);
+    @include border-radius(4px);
+    top: 5px;
+    width: 100%;
+    background: $white-t1;
+    font-family: $sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    letter-spacing: normal;
+    &.is-active {
+      background: $white;
+    }
+  }
+
+  .search-button, .cancel-button {
+    @include box-sizing(border-box);
+    display: block;
+    position: absolute;
+    top: 0;
+    @include right(12px);
```

Comment:
I would order all extends and includes at the top of your property rule listing, which helps make overriding rules easier to spot. this include should be up with box-sizing.